app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php

```diff
@@ -41,4 +41,35 @@ public function insertException($reportData) {
 
         $this->client->notifyOnError($reportData[0],$backtraces);
     }
+
+    /**
+     * @param string $message
+     */
+    public function sendToAirbrake($message) {
+        $backtraceArray = explode("\n", $message);
+        if (count($message) < 1) {
```

Comment:
It was intended as a sanity check, so we don't push empty messages to the api. 
Usually $message will contain one blank line at the beginning (AFAIK it's added by Magento), so $message will contain at last a "\n".
This is why I was checking after explode. You're right, it should be count ( $backtraceArray) instead.
I will push an update.